packages/jaeger-ui/src/components/QualityMetrics/index.tsx

```diff
@@ -92,9 +92,10 @@ export class UnconnectedQualityMetrics extends React.PureComponent<TProps, TStat
     this.setState({ loading: true });
 
     JaegerAPI.fetchQualityMetrics(service, lookback)
-      .then((qualityMetrics: TQualityMetrics) => {
-        this.setState({ qualityMetrics, loading: false });
+      .then((qualityMetrics: { data: TQualityMetrics }) => {
```

Comment:
I don't understand why we need to be making a breaking change to the data format? The current code does not expect `data:` element. The backend code does not require it either.